code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -354,9 +354,14 @@
 	explanation_text = "Steal all"
 
 /datum/objective/abductee/steal/New()
-	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap"))
+	var/target = pick(list("pets","lights","monkeys","fruits","shoes","bars of soap", "weapons", "computers", "organs"))
 	explanation_text+=" [target]."
 
+/datum/objective/abductee/paint
+	explanation_text = "The station is hideous. You must color it all"
+	var/color = pick(list("red", "blue", "green", "yellow", "orange", "purple", "black", "in rainbows"))
+	explanation_text+= "[color]!"
```

Comment:
Probably just do the colour picking before the line and have the explanation text all in one line. Considering it's fucking with your compile.